README.md

```diff
@@ -1,37 +1,48 @@
 # Mattermost
 
-Mattermost Team Edition is an open source, self-hosted Slack-alternative. 
+Mattermost is an open source, self-hosted Slack-alternative from [https://mattermost.org](https://mattermost.org).
 
-It's written in Golang and React and runs as a single Linux binary with either MySQL or Postgres. Every month on the 16th a new compiled version is released under an MIT license. 
+It's written in Golang and React and runs as a single Linux binary with MySQL or Postgres. Every month on the 16th [a new compiled version is released under an MIT license](https://www.mattermost.org/download/).
```

Comment:
Yes, but I think we only 'officially' support linux right now (hence all our install guides being for linux distros)